lib/broken_record/job.rb

```diff
@@ -17,7 +17,7 @@ def perform
           record_ids.each_slice(batch_size) do |id_batch|
             model_scope.where("#{klass.table_name}.#{primary_key}" => id_batch).each do |r|
               begin
-                if !r.valid?
+                if !r.valid?(:broken_record)
```

Comment:
Can we pass multiple contexts here so we don't break the existing behavior of testing `:update` validations?